apps/rig/lib/rig/config.ex

```diff
@@ -161,4 +186,21 @@ defmodule Rig.Config do
       {host, String.to_integer(port)}
     end)
   end
+
+  # ---
+
+  defp resolve_path(path) do
+    :code.priv_dir(:rig) |> Path.join(path)
+  end
+
+  # ---
+
+  defp disable_https(_) do
```

Comment:
This is maybe too much, but I see your thinking.

Reply:
@mmacai 

I did this because i think `update_in/3` requires a function to be passed...

```
      # DISABLE HTTPS
      config
      |> update_in([:https], &disable_https/1)
```
I'm not sure if there's a better way to use that function?!

Reply:
Perhaps this should work `fn _ -> false end`, but maybe this one is better, since thanks to function name you know what's going on, so we can keep it (doesn't matter that much at the end of the day).

Reply:
aah - true that :) OK. I'll keep like that